<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>18</red>
        <green>22</green>
        <blue>36</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>240</red>
        <green>240</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>18</red>
        <green>22</green>
        <blue>36</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>端口转发</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTableWidget" name="tableWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>200</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QTableWidget{ color:black;
	background-color:rgb(128,128,128);
	selection-color:rgb(0, 0, 0);
	selection-background-color:rgb(	100,149,237);
	border:0px;
	font: 8pt 'Microsoft YaHei';
}
QHeaderView::section:horizontal{
	border: 0px solid rgb(255, 255, 255);
	border - bottom: 0px;
	color: rgb(255, 255, 255);
	background: rgb(128, 0, 128);
	padding - left: 2px;
	font: 700 14pt 'Microsoft YaHei UI';
	min - width:60px;
}</string>
      </property>
      <column>
       <property name="text">
        <string>1</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>3</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>4</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>5</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonAdd">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	font: 700;
	font-size:35px;
	color:white;
	border-radius:25px;
    border-width:3px;
	border-style:solid;
	border-color: white;
	background-color:transparent;
}
QPushButton:hover {
	font: 700;
	font-size:35px;
	color: rgb(128, 128, 128);
	border-radius:25px;
    border-width:3px;
	border-style:solid;
	background-color:white;
}
QPushButton:pressed {
	font: 700;
	font-size:35px;
	color:white;
	border-color:white;
    border-width:0px;
	border-style:solid;
	background-color:rgb(128, 128, 128);
}</string>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonDel">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	font: 700;
	font-size:35px;
	color:white;
	border-radius:25px;
    border-width:3px;
	border-style:solid;
	border-color: white;
	background-color:transparent;
}
QPushButton:hover {
	font: 700;
	font-size:35px;
	color: rgb(128, 128, 128);
	border-radius:25px;
    border-width:3px;
	border-style:solid;
	background-color:white;
}
QPushButton:pressed {
	font: 700;
	font-size:35px;
	color:white;
	border-color:white;
    border-width:0px;
	border-style:solid;
	background-color:rgb(128, 128, 128);
}</string>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonOk">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	font: 700;
	font-size:35px;
	color:white;
	border-radius:25px;
    border-width:3px;
	border-style:solid;
	border-color: white;
	background-color:transparent;
}
QPushButton:hover {
	font: 700;
	font-size:35px;
	color: rgb(128, 128, 128);
	border-radius:25px;
    border-width:3px;
	border-style:solid;
	background-color:white;
}
QPushButton:pressed {
	font: 700;
	font-size:35px;
	color:white;
	border-color:white;
    border-width:0px;
	border-style:solid;
	background-color:rgb(128, 128, 128);
}</string>
        </property>
        <property name="text">
         <string>√</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonCancel">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	font: 700;
	font-size:35px;
	color:white;
	border-radius:25px;
    border-width:3px;
	border-style:solid;
	border-color: white;
	background-color:transparent;
}
QPushButton:hover {
	font: 700;
	font-size:35px;
	color: rgb(128, 128, 128);
	border-radius:25px;
    border-width:3px;
	border-style:solid;
	background-color:white;
}
QPushButton:pressed {
	font: 700;
	font-size:35px;
	color:white;
	border-color:white;
    border-width:0px;
	border-style:solid;
	background-color:rgb(128, 128, 128);
}</string>
        </property>
        <property name="text">
         <string>X</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonStart">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	font: 700;
	font-size:35px;
	color:white;
	border-radius:25px;
    border-width:3px;
	border-style:solid;
	border-color: white;
	background-color:transparent;
}
QPushButton:hover {
	font: 700;
	font-size:35px;
	color: rgb(128, 128, 128);
	border-radius:25px;
    border-width:3px;
	border-style:solid;
	background-color:white;
}
QPushButton:pressed {
	font: 700;
	font-size:35px;
	color:white;
	border-color:white;
    border-width:0px;
	border-style:solid;
	background-color:rgb(128, 128, 128);
}</string>
        </property>
        <property name="text">
         <string>X</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
